app/models/line_item.rb

```diff
@@ -0,0 +1,4 @@
+class LineItem < ActiveRecord::Base
+  belongs_to :cart
+  belongs_to :product
```

Comment:
Maybe we could validate the presence of `cart` and `product`? Also, it'd be great to validate quantity.

Reply:
Need to talk to you first.